modules/openapi-generator/src/main/resources/Java/libraries/apache-httpclient/ApiClient.mustache

```diff
@@ -274,23 +271,6 @@ public class ApiClient{{#jsr310}} extends JavaTimeFormatter{{/jsr310}} {
     return this;
   }
 
-  /**
-   * Gets the status code of the previous request
-   *
-   * @return Status code
-   */
-  public int getStatusCode() {
-    return statusCode;
-  }
-
-  /**
-   * Gets the response headers of the previous request
-   * @return Response headers
-   */
-  public Map<String, List<String>> getResponseHeaders() {
-    return responseHeaders;
-  }
-
```

Comment:
to make this PR backward compatible, can we keep these changes and mark these methods as deprecated instead?

Reply:
Yes, I've re-added these methods.  The implementations are a bit more complex now, but they should be thread safe.